spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/mongo/MongoProperties.java

```diff
@@ -105,6 +106,8 @@ public class MongoProperties {
 	 */
 	private Boolean autoIndexCreation;
 
+	private String additionalHosts;
```

Comment:
I think we should use a `List<String>` here so that yaml list parsing will work.

Reply:
Agreed